### PR TITLE
Feature/rifex-32/Domains-Page

### PR DESCRIPTION
**Pull request by rorolopetegui**
This implements the feature to show all domains that a user has. 

- Mocks domains and shows them
- Implements search bar, when selecting one, it goes to detail (PR: #4 )

At this moment the domains are displayed like this: 
Domains

This receives a Json with data, and the color is assigned at the moment it's processed.

### CODE REVIEW

File: old-ui/css.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ var cssFiles = {
   'react-css': fs.readFileSync(path.join(__dirname, '..', 'node_modules', 'react-select', 'dist', 'react-select.css'), 'utf8'),
   'dropdowns.css': fs.readFileSync(path.join(__dirname, '/app/css/dropdowns.css'), 'utf8'),
   'app-bar.css': fs.readFileSync(path.join(__dirname, '/app/css/app-bar.css'), 'utf8'),
+  'rif.css': fs.readFileSync(path.join(__dirname, '/app/css/rif.css'), 'utf8'),
```
Comment (marcosmartinez7) [inline old-ui/css.js]: I like this!

File: ui/app/rif/pages/domainsPage.js [hunk 2]
```diff
@@ -1,46 +1,81 @@
...
+  }
+  chiplet = (data) => {
+	return <div id="chiplet" className={'chiplet'}>
+		<div className={'chiplet-body'}>
+			<div id="chipletTitle" className={'chiplet-title'}>
+				{data.domain}
+			</div>
+			<div id="chipletDescription" className={'chiplet-description'}>	
+				<div id="chipletExpiration">
+					Expiration: {data.expiration}
```
Comment (marcosmartinez7) [inline ui/app/rif/pages/domainsPage.js]: "Expiration" should be on a span

File: ui/app/rif/pages/domainsPage.js [hunk 3]
```diff
@@ -1,46 +1,81 @@
...
+				</div>
+				<div id="chipletRenew">
+					Auto-renew: {data.autoRenew ? 'on' : 'off'}
```
Comment (marcosmartinez7) [inline ui/app/rif/pages/domainsPage.js]: Same here

File: ui/app/rif/pages/domainsPage.js.test [hunk 4]
```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
```
Comment (marcosmartinez7) [inline ui/app/rif/pages/domainsPage.js.test]: What is the idea of this js file and why is exactly the same as the `domainsPage.js`?

**Review by marcosmartinez7 — CHANGES_REQUESTED** [review]

File: ui/app/rif/pages/domainsPage.js.test (hunk 4 above)
Comment (rorolopetegui) [reply ui/app/rif/pages/domainsPage.js.test]: It was just to not loose code that i need... It's deleted now.

File: old-ui/app/css/rif.css [hunk 5]
```diff
@@ -0,0 +1,65 @@
+.body {
```
Comment (JONAF2103) [inline old-ui/app/css/rif.css]: This needs to be in a file inside a folder called rif, something like old-ui/app/css/rif/feature-name.css or something like that, if we start to add css styles to only one file by the time we end working on this we will have a huge file of css with a lot of lines. It's hard to maintain, we should put that in different files.

File: old-ui/css.js (hunk 1 above)
Comment (JONAF2103) [inline old-ui/css.js]: Taking the comment above, here we should read a folder instead.

File: package.json [hunk 6]
```diff
@@ -179,6 +179,7 @@
     "raven-js": "^3.24.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-dropdown-select": "^4.3.0",
```
Comment (JONAF2103) [inline package.json]: This is necessary? i mean why we are adding a new library for a dropdown, we don't have any?

**Review by JONAF2103 — CHANGES_REQUESTED** [review]

File: ui/app/rif/pages/domainsPage.js [hunk 7]
```diff
@@ -1,49 +1,104 @@
...
-              h('div', 'RNS Domains'),
-            ]),
-          ]),
-        ]),
-      ])
-    )
+	return (
+	  <div className={'body'}>
+		<FontAwesomeIcon icon={faChevronLeft} className={'rif-back-button'} onClick={() => this.props.goHome()}/>
+		<Select
```
Comment (marcosmartinez7) [inline ui/app/rif/pages/domainsPage.js]: Why are we using a Select instead of a normal search bar?

**Review by marcosmartinez7 — CHANGES_REQUESTED** [review]
I agree to make the changes requested by @JONAF2103

File: package.json (hunk 6 above)
Comment (rorolopetegui) [reply package.json]: It's a component that we can use for multiple purposes, its very used and have lots of functionalities. I think we should use it.
https://www.npmjs.com/package/react-dropdown-select

File: ui/app/rif/pages/domainsPage.js (hunk 7 above)
Comment (rorolopetegui) [reply ui/app/rif/pages/domainsPage.js]: Its the name of the component. 
https://www.npmjs.com/package/react-dropdown-select

File: old-ui/app/css/rif.css (hunk 5 above)
Comment (rorolopetegui) [reply old-ui/app/css/rif.css]: You right, i'm working on this!

File: old-ui/css.js (hunk 1 above)
Comment (rorolopetegui) [reply old-ui/css.js]: You right, i'm working on this!

File: old-ui/app/css/rif.css (hunk 5 above)
Comment (rorolopetegui) [reply old-ui/app/css/rif.css]: Check the new structure, as fs cant import a directory on windows/mac we need to import each .css individually.

File: old-ui/css.js (hunk 1 above)
Comment (rorolopetegui) [reply old-ui/css.js]: Please, check the comment above

File: ui/app/rif/components/searchDomains.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+		if(localStorage.rnsDomains){
+			let domains = JSON.parse(localStorage.rnsDomains);
+			this.state = { 
+				domains: domains,
+			}
+		}	
+	}
+  render () {
+	return (
+		<Select
```
Comment (marcosmartinez7) [inline ui/app/rif/components/searchDomains.js:22]: Why are we using a Select instead of a searchbar?

File: package.json (hunk 6 above)
Comment (marcosmartinez7) [reply package.json]: We already have dropdown menus, at least on the contract interaction page (for selecting the contract method). So i will not add a new library for that. 

Also, the design specifies a searchbar nor a dropdown item

**Review by marcosmartinez7 — APPROVED** [review]
LGTM

**Review by marcosmartinez7 — APPROVED** [review]

**Review by JONAF2103 — APPROVED** [review]
LGTM

**Pull request merged**